### PR TITLE
remove --preview flag and update extension laoding

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs [hunk 1]
```diff
@@ -1,183 +1,190 @@
...
+{
+    public static void  Load(Kernel kernel)
+    {
+        if (kernel.RootKernel is CompositeKernel compositeKernel)
+        {
+            var httpRequestKernel = new HttpRequestKernel();
+            compositeKernel.Add(httpRequestKernel);
+            httpRequestKernel.UseValueSharing();
+            
+            Formatter.Register<HttpResponseMessage>((responseMessage, context) =>
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs:28]: Are these not redundant with the ones already provided?

File: src/Microsoft.DotNet.Interactive.HttpRequest/extension.dib [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#!meta
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.HttpRequest/extension.dib:1]: The metadata should be left out for this.

File: src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs (hunk 1 above)
Comment (colombod) [reply src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs]: removed

File: src/Microsoft.DotNet.Interactive.HttpRequest/extension.dib (hunk 2 above)
Comment (colombod) [reply src/Microsoft.DotNet.Interactive.HttpRequest/extension.dib:1]: ok

File: src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs [hunk 3]
```diff
@@ -1,183 +1,151 @@
...
+                Added kernel `{httpRequestKernel.Name}`. Send HTTP requests using the following syntax:
+
+                ```
+                GET https://example.com
+                ```
+                """, "text/markdown");
+        }
+    }
+
+    private static async Task FormatHttpResponseMessageAsPlainText(
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs:38]: Doesn't this now duplicate the one in the `Formatting` package?

File: src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs [hunk 4]
```diff
@@ -1,183 +1,151 @@
...
+                margin: 0 .5em;
+            }}
+
+            .{_containerClass} summary, .{_logContainerClass} summary {{
+                margin: 1em 0;
+                font-size: 1.17em;
+                font-weight: 700;
+            }}");
+
+    private static async Task FormatHttpResponseMessageAsHtml(
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs:102]: Ditto.

File: src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs (hunk 3 above)
Comment (colombod) [reply src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs:38]: OUCH

File: src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs (hunk 4 above)
Comment (colombod) [reply src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs:102]: reducing carbon footprint now

**Pull request merged**